README.md

```diff
@@ -63,15 +65,14 @@ and instructions on how to update it are in the [`README.md`](https://github.com
 
 ### Updating CSS for the 2018 Theme
 
-The website for 2018 uses sass to compile all the assets it has, in
-order to modify the theme first you must install the dependencies of
-gulp, you can do that by running in the main directory:
+The website for 2018 uses SASS to compile all the assets it has. To modify the theme, you must first install the dependencies of
+`gulp`. In the main directory, run:
```

Comment:
Looks like I modified the line formatting here? Does this need to be redone?

Reply:
In terms of output, those two are the same.

There is some 'discussion' going on as to whether we should fully adopt a single sentence per line (probably with a soft wrap set in your editor), a hard character limit or some other arbitrary rule :-p.